src/main/java/de/dennisguse/opentracks/viewmodels/StatsData.java

```diff
@@ -0,0 +1,56 @@
+package de.dennisguse.opentracks.viewmodels;
+
+
+import android.util.Pair;
+
+import androidx.annotation.NonNull;
+
+public class StatsData {
```

Comment:
feels like overlapping with Field

Reply:
Yes, I refactored it.

Now we have:
- StatisticsField that has the following attributes: key, title, isVisible, isPrimary and isWide.
- Layout with several StatisticsFields.
- StatisticsData that has the following attributes: StatisticsField, value, unit, (optional) description.

Basically:

```
class StatisticsField {
    - key
    - title
    - isVisible
    - isPrimary
    - isWide
}

class StatisticsData {
    - value
    - unit
    - description
}
StatisticsData o-- StatisticsField

class Layout {
    - profile
}
Layout "1" o-- "many" StatisticsField
```

PS/ I'm going to update `data-structure.txt` with all this information.